mne/stats/regression.py

```diff
@@ -392,3 +392,254 @@ def _make_evokeds(coefs, conds, cond_length, tmin_s, tmax_s, info):
             kind='average')  # nave and kind are technically incorrect
         cumul += tmax_ - tmin_
     return evokeds
+
+
+class EncodingModel(object):
+    def __init__(self, delays=None, est=None, preproc_y=True):
+        """Fit a STRF model.
+
+        Fit a receptive field using time lags and a custom estimator or
```

Comment:
Probably much more general than "fit a receptive field" no?

"Fit a neural response (e.g., a receptive field) and ..."